core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ExecutionServiceCluster.scala

```diff
@@ -26,6 +26,7 @@ trait ExecutionServiceCluster extends ErrorReportable {
                       userInfo: UserInfo
   ): Future[(ExecutionServiceId, Seq[Either[ExecutionServiceStatus, ExecutionServiceFailure]])]
 
+  // currently unused
   def status(workflowRec: WorkflowRecord, userInfo: UserInfo): Future[ExecutionServiceStatus]
```

Comment:
I chose not to delete the now-unused `status()` method, but I could be convinced it needs cleaning up

Reply:
shrug, why do you want to keep it?